### DISCUSSION TITLE
Why I have'nt icons such as "Rofi", "Microphone", "Audio" etc. on my waybar?

### DISCUSSION THREAD

**Discussion by Dolbaeb777K** [JaKooLit/Hyprland-Dots#179] [Q&A] [answered]
Sometimes they appear, but rarely.
 
![image](https://github.com/JaKooLit/Hyprland-Dots/assets/126266582/59bf9a37-18a5-4ad5-b128-acfc7c51f057)

although there is a normal penguin sound and microphone in the `modules` file...
![image](https://github.com/JaKooLit/Hyprland-Dots/assets/126266582/b8560986-ed91-4522-8b93-b565e066f002)

**Comment by JaKooLit** [comment]
hmm...

any updated on this?

**Reply by Dolbaeb777K** [reply]
Yes. I found and installed JetBrainMono font to the `/home/hippoid/.local/share/fonts directory/` from the https://github.com/wangrunlin/nerd-font-JetBrains-windows.git

Now, I have very small icons, but no matter how much I tried to solve it, I decided to put up with it....
![image](https://github.com/JaKooLit/Hyprland-Dots/assets/126266582/c0911991-8483-4796-9c72-041580e070e8)

**Reply by JaKooLit** [reply]
![Screenshot_25-Apr_19-33-07_15524](https://github.com/JaKooLit/Hyprland-Dots/assets/85185940/1c107d87-cf79-4495-9370-3e1a119c149f)


you can increase the font scaling and everything should increase 

or edit the waybar css

by default, its 97%. You can increase to 100% or even 105% if you prefer

**Comment by Dolbaeb777K** [comment]
I've tried this, unfortunately, it enlarges the whole thing, but it doesn't affect the `rofi`, `volume`, `microphone` and etc. icons in any way.
Variant with waybar css:
![image](https://github.com/JaKooLit/Hyprland-Dots/assets/126266582/76fad4fb-d0de-426c-bdbf-b66fb542dcc0)

Variant with nwg-look:
![image](https://github.com/JaKooLit/Hyprland-Dots/assets/126266582/1c2fc977-d6cf-4233-9675-e7c5d937c0c8)
text scaling isn't comfortable variant, because it scaling anything..

In the your video installation Hyprland on opensuse, I can see the normal icon's size. Maybe it's the font itself, then where did you get your font that everything works properly?
![image](https://github.com/JaKooLit/Hyprland-Dots/assets/126266582/ef73c181-afc5-46a3-b32e-faf679c02f6a)

**Reply by JaKooLit** [reply]
rofi is different config

remember that on my presentations, I dont use monitor scaling. Even my systems. I only use font scaling on my 2k 15 inches monitor since it all looks small.

all fonts are on the script

**Reply by Dolbaeb777K** [reply]
If all fonts is installed by scrypt, but why  Is `JetBrainsMono` present in style.css, but missing from the system? Even in the `install-31-135256_fonts` and in `fonts.sh` He's not here.
![Screenshot_25-апр_19-15-06_4651](https://github.com/JaKooLit/Hyprland-Dots/assets/126266582/c49b71b8-f2b2-4300-a4f0-ab8e68caf9b5)

**Reply by JaKooLit** [reply, marked as answer]
https://github.com/JaKooLit/Fedora-Hyprland/blob/main/install-scripts%2Ffonts.sh#L40

starts here

**Reply by Dolbaeb777K** [reply]
IT WORKS!! Thank you! Very mush!
So, opensuse's `fonts.sh` doesn't have `JetBrainsMono` install. You are going to release a global update of your `Hyprland-dots`, so don't forget to include this `font` in it so that such `incidents` do not arise ))
![image](https://github.com/JaKooLit/Hyprland-Dots/assets/126266582/79534a18-d256-47e7-b181-77fea1a4fe2a)

**Reply by JaKooLit** [reply]
https://github.com/JaKooLit/OpenSuse-Hyprland/commit/dd408b969a5afa47a531d34535ef07ab023f4ce1
